src/Adapter/ImageThumbnailsRegenerator.php

```diff
@@ -0,0 +1,310 @@
+<?php
```

Comment:
I wonder if this file doesn't need to be simplified. I think that too much code is imbricated.

Reply:
I agree with you, but this class is aim to be deleted when we have a new system to create images thumbnails.
So for now, i've just take some old stuff from old AdminImagesController.